Cpp/Library/Control/SingleRunController.cc

```diff
@@ -36,12 +38,125 @@ namespace Nymph
 
         Controller::Configure( node );
 
+        // The run queue is an array of processor names, or groups of names, which will be run sequentially.
+        // If names are grouped (in another array), those in that group will be run in parallel.
+        if( ! node.has("run-queue") )
+        {
+            LWARN( contlog, "Run queue was not specified" );
+        }
+        else
+        {
+            ConfigureRunQueue( node["run-queue"].as_array() );
+        }
+
+        return;
+    }
+
+    void SingleRunController::ConfigureRunQueue( const scarab::param_array& array )
```

Comment:
Is this where the ordering of the processors gets arranged (e.g. series or parallel)?

Reply:
That's right.  The configuration could look something like this: `[proc1, proc2]` to run those two in series.  Or if it were configured with `[[proc1, proc2], proc3]`, then `proc1` and `proc2` would run in parallel, and then `proc3` would follow in series.

Reply:
This reminds me that I forgot to include the documentation.  That also needs to move over from ProcessorToolbox.